profiler/src/ProfilerEngine/Datadog.Profiler.Native/StackSamplerLoop.cpp

```diff
@@ -329,7 +329,7 @@ void StackSamplerLoop::CollectOneThreadStackSample(
     PROFILING_TYPE profilingType)
 {
     HANDLE osThreadHandle = pThreadInfo->GetOsThreadHandle();
-    if (osThreadHandle == static_cast<HANDLE>(0))
+    if (osThreadHandle == static_cast<HANDLE>(nullptr))
```

Comment:
I'm not sure HANDLE should be seen as a pointer: most of the time it is an offset in a table so 0 makes more sense than nullptr